configure.acr

```diff
@@ -52,6 +52,9 @@ IFAND HAVE_LIB_Z USE_ZIP {
 	LIBZIP = '' ; (( static linking to shlr/zip ))
 }
 
+HAVE_LIB_XXHASH = 0 ;
```

Comment:
you must add an option to let the user choose if they want to use the xxhash from r2 even if they have the lib in the system. imho i would go for --with-sys-xxhash, because i think that not depending on anything even if found makes the software more predictible